hashes/src/internal_macros.rs

```diff
@@ -142,7 +142,7 @@ pub(crate) use hash_trait_impls;
 /// The `from_engine` free-standing function is still required with this macro. See the doc of
 /// [`hash_trait_impls`].
 macro_rules! hash_type {
-    ($bits:expr, $reverse:expr, $doc:literal) => {
+    ($mod:ident, $bits:expr, $reverse:expr, $doc:literal) => {
```

Comment:
This shouldn't be needed, the macro is only ever called in the module itself which contains `HashEngine`.

Reply:
Ah, of course. Thanks man, I'll put a bit more effort in. Thanks.